Makefile

```diff
@@ -15,3 +15,10 @@ install:
 	install -d ${PREFIX}/share/examples/snare
 	install -c -m 444 snare.conf.example ${PREFIX}/share/examples/snare
 
+distrib:
+	test "X`git status --porcelain`" = "X"
+	@read v?'snare version: '; mkdir snare-$$v; \
+      cp -rp Cargo.lock Cargo.toml COPYRIGHT LICENSE-APACHE LICENSE-MIT \
+	    Makefile README.md build.rs snare.1 snare.conf.5 snare.conf.example \
+		src snare-$$v ; \
```

Comment:
I suspect some of this stuff should use `&&` to only proceed on success?

Reply:
You're quite right! Fixed in https://github.com/softdevteam/snare/pull/36/commits/b6d0425cbf1b98462cb8c0e83bd0cbf8248f1b0c.